Templates/HttpTrigger-PowerShell-Modules/modules/Show-Calendar.psm1

```diff
@@ -0,0 +1,147 @@
+<# 
+ .Original source
+ https://msdn.microsoft.com/en-us/library/dd878340(v=vs.85).aspx

```

Comment:
Perhaps we can simplify this "modules" sample into a more "real world" example. The goal of the sample is only to show how to allow a module to be required - therefore it makes sense to make the module as simple as possible.